decidim-admin/spec/features/manage_participatory_processes_spec.rb

```diff
@@ -37,12 +37,12 @@
       expect(page).to have_content(participatory_process.subtitle["en"])
       expect(page).to have_content(participatory_process.subtitle["es"])
       expect(page).to have_content(participatory_process.subtitle["ca"])
-      expect(page).to have_content(participatory_process.short_description["en"])
-      expect(page).to have_content(participatory_process.short_description["es"])
-      expect(page).to have_content(participatory_process.short_description["ca"])
-      expect(page).to have_content(participatory_process.description["en"])
-      expect(page).to have_content(participatory_process.description["es"])
-      expect(page).to have_content(participatory_process.description["ca"])
+      expect(page).to have_content(participatory_process.short_description["en"].gsub(/<\/?[^>]*>/, ""))
```

Comment:
No, I'm testing that the `display_attribute` helper method works as I expect it to work (that is, showing all available languages in the `show` page).